src/server/VRxController.py

```diff
@@ -144,6 +145,13 @@ def do_startup(self,arg):
         # (Because the pi was already running, they should all be connected to the broker)
         # Even if the server.py is restarted, the broker continues to run:)
 
+    def do_shutdown(self, arg):
+        self.logger.info("VRx Control Shutting down")
+        self._seat_broadcast.clear_user_message()
+        self._seat_broadcast.turn_on_osd()
+        self._seat_broadcast.set_wifi_state(clearview.comspecs.cv_device_limits["wifi_mode_ap"])
```

Comment:
We should continue to move toward generic methods and away from requiring the CV library to run VRx Control.